src/navigator/Navigator.tsx

```diff
@@ -602,7 +605,15 @@ export function MainStackScreen() {
 
   return (
     <Stack.Navigator initialRouteName={initialRouteName} screenOptions={emptyHeader}>
-      <Stack.Screen name={Screens.DrawerNavigator} component={DrawerNavigator} options={noHeader} />
+      {getFeatureGate(StatsigFeatureGates.USE_TAB_NAVIGATOR) ? (
```

Comment:
do we have to conditionally register these screens?

Reply:
Registered both in 5e5a279b44481f32f53dd4b4f9bd411c79d6f71a!